Yandex ID System page

chernodub opened the pull request:
<!--
Thanks for improving the documentation 😃
Please give a short description and check the matching checkboxes to help us review this as quick as possible.

Please make the PR writeable. This allows us to fix typos, grammar and linting errors ourselves, which makes
merging and reviewing a lot faster for everybody.
-->

Hi! This PR accompanies the one submitted in Prebid.js repo: https://github.com/prebid/Prebid.js/pull/11196

We have added the Yandex ID system.

If any details needed, we'd be happy to provide it, thanks!

## 🏷 Type of documentation
<!-- Remove items that don't apply and/or select an item by changing [ ] to [x] -->
- [x] new id system

## 📋 Checklist
<!-- Remove items that don't apply and/or select an item by changing [ ] to [x] -->
- [x] Related pull requests in prebid.js or server are linked -> Paste link in this list or reference it on the PR itself

chernodub commented:
I am not sure why, but the module did not appear on `/download.html`. I would appreciate it if somebody would let me know what I missed. Thanks :)

bretg reviewed (changes requested):
Sorry, just realized that you also need to add a row into the table at https://docs.prebid.org/dev-docs/modules/userId.html#prebidjs-adapters

chernodub commented:
Hi @bretg! I've updated the file according to your recommendation.

I've also updated the “new user ID module” checklist according to your comment.

Please let me know if anything else needed. Thanks!

bretg commented:
Thanks. 

> the module did not appear on /download.html

Not sure if this comment is old, but I see Yandex ID on the preview -- https://deploy-preview-5268--prebid-docs-preview.netlify.app/download


Note that the linting error here comes from a problem not related to this PR and I don't know how to fix... the linter is confused by the liquid script containing a pipe symbol. For now, ignoring the lint issue. We'll merge these docs when the code is released.

chernodub commented:
@bretg yep! The download page worked as expected after I updated the adapters table. Thanks a lot!

muuki88 commented:
@chernodub can you resolve the merge conflicts?

chernodub commented:
@muuki88 thanks! I've made the updates. Please take a look when you can :)

muuki88 commented:
Awesome 😎
